tests/cpp/operator/mkldnn_test.cc

```diff
@@ -100,7 +100,7 @@ static void VerifyDefMem(const mkldnn::memory &mem) {
 
 TEST(MKLDNN_UTIL_FUNC, MemFormat) {
   // Check whether the number of format is correct.
-  CHECK_EQ(mkldnn_format_tag_last, 131);
+  CHECK_EQ(mkldnn_format_tag_last, 152);
```

Comment:
Where can we get this number? Or do we need to calculate this number manually when upgrading to a new version of DNNL?

Reply:
We need print this flag in cpp to get the value. This test was design to notify us if there are new formats added to dnnl so we need carefully check if these new formats break any integration code or the integration code need be improved to handle them.